File: src/main/java/org/dxworks/githubminer/dto/response/repository/refs/RefObject.java
package org.dxworks.githubminer.dto.response.repository.refs;

import com.google.api.client.json.GenericJson;
import com.google.api.client.util.Key;
import lombok.Data;

@Data
public class RefObject extends GenericJson {
    @Key
    private String type;
    @Key
    private String sha;
}

File: github-miner-core/src/main/kotlin/org/dxworks/githubminer/dto/response/repository/refs/RefObject.kt
package org.dxworks.githubminer.dto.response.repository.refs

import com.google.api.client.json.GenericJson
import com.google.api.client.util.Key

class RefObject : GenericJson() {
    @Key
    var type: String? = null

    @Key
    var sha: String? = null
}
